src/scripts/dungeons/Dungeon.ts

```diff
@@ -8154,7 +8154,9 @@ dungeonList['Mistralton Cave'] = new Dungeon('Mistralton Cave',
     [
         new DungeonBossPokemon('Drilbur', 23000000, 100),
         new DungeonBossPokemon('Axew', 24000000, 100),
-        new DungeonBossPokemon('Cobalion', 25000000, 100),
+        new DungeonBossPokemon('Cobalion', 25000000, 100, {
+            requirement: new QuestLineCompletedRequirement('Swords of Justice', GameConstants.AchievementOption.more),
```

Comment:
along with adding a step for catching them, this needs changed to require the current final step of the quest, as do Terrakion and Virizion.
Also, GameConstants.AchievementOption.more is the default, so it's unnecessary, Same goes for all other uses of that within the PR

Reply:
changed the requirements to match it